src/web/common.rkt

```diff
@@ -67,44 +67,135 @@
                               ,@(for/list ([line extra])
                                   `(li ,line)))))))))
 
+(define (render-comparison #:title [title #f] name a b )
+  (render-large #:title title name a `(span ([class "unit"]) " → ") b))
+
 (define (render-large #:title [title #f] name . values)
   `(div ,name ": " (span ([class "number"]
                           ,@(if title `([title ,title]) '()))
                          ,@values)))
 
-(define (render-comparison #:title [title #f] name a b )
-  (render-large #:title title name a `(span ([class "unit"]) " → ") b))
+(define (preprocess-sort-core output sort-preprocesses)
```

Comment:
All of these are super repetitive, which also makes them hard to read. Make each of these take in a _single_ preprocess instruction, and output a string. Don't make it specific to `sort`: pass in the whole preprocess instruction. Call them something like `preprocess->xxx`. Then we can have a separate function to combine these together with the actual code; I think we'll only need a special-case for TeX and maybe FPCore, everything else can just concatenate them with newlines.